rnd/autogpt_builder/src/api/autogpt_server_api.ts

```diff
@@ -1,5 +1,4 @@
-import { XYPosition } from "reactflow";
-import { ObjectSchema } from "./types";
+import {Block, NodeExecutionResult} from "@/types/api";
```

Comment:
All types are stripped from this file and only 2 imported back in:
![image](https://github.com/Significant-Gravitas/AutoGPT/assets/12185583/9da69f0b-d2b2-445d-9be8-5184897c93fb)

We really need pre-commit and CI checks on this subproject.